src/install/local/remove.md

```diff
@@ -0,0 +1,46 @@
+# Remove MDAI Engine
```

Comment:
_Destroy_ or _Uninstall_ vs. _Remove_ feels better and more technical. You're destroying it and it's dependencies during our uninstall process

Reply:
refactored in the latest commit.